setup.py

```diff
@@ -28,9 +28,9 @@
     print(error, file=sys.stderr)
     sys.exit(1)
 
-from distutils.core import setup
+from setuptools import setup
 
-import distutils.dir_util
+import setuptools
```

Comment:
actually, this could have been untouched as only thing it is used is to clean build folder.
Alternatively it could be ```from setuptools import distutis.dir_util```.